src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -25,16 +25,16 @@
 import java.util.Objects;
 import java.util.UUID;
 
+import io.vertx.core.json.JsonObject;
 import org.apache.commons.lang3.StringUtils;
 import org.folio.circulation.domain.policy.LoanPolicy;
 import org.folio.circulation.domain.representations.LoanProperties;
 import org.folio.circulation.support.Result;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 
-import io.vertx.core.json.JsonObject;
-
 public class Loan implements ItemRelatedRecord, UserRelatedRecord {
+  private static final String DECLARED_LOST_ACTION_NAME = "declaredLost";
```

Comment:
Where are the other actions declared? Could this be kept with them?

Reply:
Other acitons are magic strings in the Loan class.

Reply:
Sure, the others are used in situ.

I think we should either use each in situ, or extract a new class / enum for the actions that are allowed.

Reply:
I have extracted a separate class for it.